ugwumadu116/meal-booking
# feature(confirm.html, order.html): update readme file

#### What does this PR do?
- adds confirmation page
- adds user order page
#### Description of Task to be completed?
- create a page where the user can see the success page after payment
- create a page where the user can see order history
#### How should this be manually tested?
- after payment is successful you will be able to navigate the success page
- from the user page click on my orders to see the history of all your orders
#### What are the relevant pivotal tracker stories?
[Delivers #163928466]
[Delivers #163928296]
